tests/Satooshi/ProjectTestCase.php

```diff
@@ -7,7 +7,7 @@ class ProjectTestCase extends \PHPUnit_Framework_TestCase
     protected function setUpDir($projectDir)
     {
         $this->rootDir       = realpath($projectDir . '/prj');
-        $this->srcDir        = $this->rootDir . '/files';
+        $this->srcDir        = realpath($this->rootDir . '/files');
```

Comment:
Are '/files' and '/prj' OK? Shouldn't we use DIRECTORY_SEPARATOR here and there?